site/docusaurus.config.js

```diff
@@ -18,6 +18,17 @@ const config = {
   projectName: 'flow',
   trailingSlash: true,
 
+  plugins: [
+    [
+      require.resolve('docusaurus-lunr-search'), 
+      {
+        // @ts-ignore
+        excludeRoutes: [
+            'blog/**/*',
+        ]
+    }]
+  ],
+
```

Comment:
Enabling the new plugin. Also - for now I think excluding the blog is best as it is being served and the search made it easy to get to on accident.

I also just ignored the type checking stuff for this library as it does not have anything type related  I could find.